KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyPresenter.swift

```diff
@@ -94,7 +95,24 @@ final class KarhooLoyaltyPresenter: LoyaltyPresenter {
             return
         }
         
-        // TODO: Finish implementing
+        // Convert $ amount to cents
+        let amount = viewModel.tripAmount * 100
```

Comment:
Unfortunately, it is not always true. 
https://en.wikipedia.org/wiki/ISO_4217

Check `CurrencyCodeConverter`